js_modules/dagster-ui/packages/ui-core/src/app/FeatureFlags.oss.tsx

```diff
@@ -4,4 +4,5 @@ export enum FeatureFlag {
   flagSidebarResources = 'flagSidebarResources',
   flagDisableAutoLoadDefaults = 'flagDisableAutoLoadDefaults',
   flagSettingsPage = 'flagSettingsPage',
+  flagCodeLocationPage = 'flagCodeLocationPage',
```

Comment:
hm kind of unfortunate but we have to duplicate this here: https://github.com/dagster-io/internal/blob/5f4d086c5a78341c9b52bd0d7e1a0d4f00f9d62a/dagster-cloud/js_modules/app-cloud/src/settings/FeatureFlags.cloud.tsx#L2

Maybe we switch from using an enum to a real object?

Reply:
Included as part of https://github.com/dagster-io/internal/pull/11260.